continunet/network/unet.py

```diff
@@ -0,0 +1,135 @@
+"""UNet model for image segmentation in keras."""
+
+import numpy as np
+
+from keras.layers import (
+    Activation,
+    BatchNormalization,
+    Concatenate,
+    Conv2D,
+    Conv2DTranspose,
+    Dropout,
+    Input,
+    MaxPooling2D,
+)
+from keras.models import Model
+from tensorflow.keras.optimizers import Adam
+
+
+class Unet:
+    """UNet model for image segmentation."""
+
+    def __init__(
+        self,
+        input_shape,
+        filters=16,
+        dropout=0.05,
+        batch_normalisation=True,
+        trained_model=None,
+        image=None,
```

Comment:
Do you not use type annotations? I find it helps enormously in an IDE for resolving what methods are available where. E.g., is input supposed to be a tuple or a single integer, makes a difference what the IDE will display when you do `input_shape.<tab>`.